ui/desktop/tests/acceptance/projects/settings/index-test.js

```diff
@@ -94,4 +101,30 @@ module('Acceptance | projects | settings | index', function (hooks) {
     await click(`[href="${urls.settings}"]`);
     assert.strictEqual(currentURL(), urls.settings);
   });
+
+  test('displays userinfo when authenticated', async function (assert) {
+    authenticateSession({ username: 'testuser' });
+    assert.expect(2);
+    await visit(urls.settings);
+    assert.ok(currentSession().isAuthenticated);
+    assert.dom(USERNAME_SELECTOR).hasText('testuser');
+  });
+
+  test('displays type of auth method when authenticated', async function (assert) {
```

Comment:
I wonder if these should be integration tests instead, since it doesn't seem like it needs the rest of the page to be tested?

And that way we can also add tests that OIDC, LDAP, and password auth methods are displayed correctly

Reply:
good point, moved to integration and added an assert for types display